crazy_rl/multi_agent/numpy/catch/catch.py

```diff
@@ -172,6 +172,7 @@ def _compute_terminated(self):
         for agent in self.agents:
             terminated[agent] = False
 
+        for agent in self.agents:
```

Comment:
why adding a loop?

Reply:
Because a part of the 2nd loop changes terminated[other_agent] so if we initialize it to False automatically it can erase this value. 
Maybe there is an other way to do it.